### PR TITLE
Incomp liqna

### PR DESCRIPTION
**Pull request by VikramGovindarajan**
### Description of the Change

A new pure fluid, liquid sodium, is added to the backend for incompressible fluids with the tabulated property data available. The applicable range of temperature is 400 K to 2500 K. JSON file with the fitted parameters is generated and added to the repository. 

### Benefits

Liquid sodium as a fluid can be imported from the CoolProp database. Liquid sodium is used as a coolant for many applications such as in fast-breeder nuclear reactors, solar thermal power plants, etc.

### Possible Drawbacks

No drawbacks are envisaged as it is only a new fluid addition.

### Verification Process

Firstly, the relative error after fitting the basic quantities (from the fitting report for liquid sodium) is very less and acceptable.

The source is then recompiled by running the following command in CoolProp/wrappers/Python/ directory,
```console
python3 all_incompressibles.py 
```

The new fluid can now be imported and used (with both high-level and low-level interface) using the following commands in the ipython interpreter:

```python
In [1]: import CoolProp
In [2]: flstate = CoolProp.AbstractState("INCOMP","LiqNa")
In [3]: flstate.update(CoolProp.PT_INPUTS,1.E5,700.)
In [4]: flstate.cpmass()
Out[1]: 1276.9047619303014

In [5]: CoolProp.CoolProp.PropsSI('D','P',1.E5,'T',700.,'INCOMP::LiqNa')
Out[2]: 846.6785937377822
```

### Applicable Issues

This change is not related to any issue.

### CODE REVIEW

File: dev/incompressible_liquids/CPIncomp/SecCoolFluids.py [hunk 1]
```diff
@@ -266,7 +266,7 @@ def getFromFile(self, data):
             for j in range(c):
                 nu = np.NAN
                 try:
-                    nu = np.float(res[i, j])
```
Comment (ibell) [inline dev/incompressible_liquids/CPIncomp/SecCoolFluids.py:269]: Please revert, not needed

File: dev/incompressible_liquids/CPIncomp/SecCoolFluids.py [hunk 2]
```diff
@@ -493,7 +493,7 @@ def getFromFile(self, data):
...
```
Comment (ibell) [inline dev/incompressible_liquids/CPIncomp/SecCoolFluids.py:496]: Please revert, not needed

**Comment by VikramGovindarajan** [comment]
When the JSON files are generated by running the script dev/incompressible_liquids/all_incompressibles.py, I get the following error if I don't change np.float() to float() in those locations in SecCoolFluids.py file:

```console
Loading SecCool fluids: Traceback (most recent call last):
  File "/mnt/e/codes/CoolProp/dev/incompressible_liquids/all_incompressibles.py", line 138, in <module>
    fluidObjs = getSecCoolFluids()
  File "/mnt/e/codes/CoolProp/dev/incompressible_liquids/CPIncomp/__init__.py", line 159, in getSecCoolFluids
    return SecCoolSolutionData.factory()
  File "/mnt/e/codes/CoolProp/dev/incompressible_liquids/CPIncomp/SecCoolFluids.py", line 298, in factory
    sec += [SecCoolSolutionData(sFile='Antifrogen KF', sFolder='xVolume', name='AKF', desc='Antifrogen KF, Potassium Formate', ref='Clariant2000,Skovrup2013', densityFactor=None, heatFactor=1e3, conductivityFactor=None, viscosityFactor=1e-3)]
  File "/mnt/e/codes/CoolProp/dev/incompressible_liquids/CPIncomp/SecCoolFluids.py", line 31, in __init__
    self.temperature.data, self.concentration.data, self.density.data = self.getArray(dataID='Rho')
  File "/mnt/e/codes/CoolProp/dev/incompressible_liquids/CPIncomp/DataObjects.py", line 326, in getArray
    fileArray = self.getFromFile(dataID)
  File "/mnt/e/codes/CoolProp/dev/incompressible_liquids/CPIncomp/SecCoolFluids.py", line 269, in getFromFile
    nu = np.float(res[i, j])
  File "/home/vikramg/.local/lib/python3.10/site-packages/numpy/__init__.py", line 305, in __getattr__
    raise AttributeError(__former_attrs__[attr])
AttributeError: module 'numpy' has no attribute 'float'.
`np.float` was a deprecated alias for the builtin `float`. To avoid this error in existing code, use `float` by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use `np.float64` here.
The aliases was originally deprecated in NumPy 1.20; for more details and guidance see the original release note at:
    https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations. Did you mean: 'cfloat'?
```

I am using a numpy version 1.24.2. From a brief web search, I understood that this error occurs following the numpy v1.24.0 release, because of the removal of numpy’s aliases for float, int, and similar dtypes.

Hence, I think it is better to modify the source code such that it’s now referencing the equivalent built-in types . i.e., to use float() instead of np.float() in those location in SecCoolFluids.py file.

**Comment by ibell** [comment]
Ok I see your point.  That does seem like the recommendation.  Let's leave
it as is in the PR and I'll merge

**Pull request merged**